src/amo/components/RatingManager/index.js

```diff
@@ -69,7 +69,10 @@ export class RatingManagerBase extends React.Component<InternalProps> {
   componentDidMount() {
     const { addon, dispatch, errorHandler, userId, userReview } = this.props;
 
-    if (errorHandler.hasError()) {
+    if (
+      errorHandler.hasError() &&
+      errorHandler.capturedError.responseStatusCode !== 429
+    ) {
```

Comment:
I think that's good enough for this edge case, however do note that technically _any_ endpoint could return a 429, even for GET requests. So I guess there is a chance the captured error here could be relevant... but we'd probably fail elsewhere on the add-on page too...

Could you add a quick comment to explain the error we're trying to work around here comes from a past attempt to post a rating ?